cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -210,6 +210,11 @@ export class Application {
     ): void {
         cy.wait(2000);
         performRowActionByIcon(this.name, editButton);
+        // cy.contains(this.name, { timeout: 120 * SEC })
+        //     .closest(trTag)
+        //     .within(() => {
+        //         click(editButton);
+        //     });
```

Comment:
comments should be removed, otherwise LGTM

Reply:
Thanks, fixed